src/hooks/useMediaQuery/useMediaQuery.playwright.tsx

```diff
@@ -13,24 +13,24 @@ test('Should detect initial screen size', async ({ page, mount }) => {
 });
 
 test('Should detect screen size changes', async ({ page, mount }) => {
-  const component = await mount(<Component />, {});
+  await mount(<Component />, {});
 
   await page.setViewportSize({
     width: 250,
     height: 250,
   });
 
   // Wait for the resize to be handled by the hook
-  await page.waitForTimeout(32);
-  await expect(component).toContainText('small');
+  await page.getByText('small').waitFor({
```

Comment:
Don't you want to still have an expect, but in a form like this:
```
await expect(page.getByText('small')).toBeVisible()
```

It will auto-wait (as most/all locators will do), and will give a proper error when it's not there.